docs/source/app-dev/ledger-api.rst

```diff
@@ -52,6 +52,22 @@ For full information on the services see :doc:`/app-dev/services`.
 
 You may also want to read the :doc:`protobuf documentation </app-dev/grpc/proto-docs>`, which explains how each service is defined as protobuf messages.
 
+How to Access the Ledger API
+****************************
+
+You can access the Ledger API via the :doc:`Java Bindings <bindings-java/index>`.
+
+If you don't use a language that targets the JVM you can use gRPC to generate the code to access the Ledger API in
+several supported programming languages. :doc:`Further documentation <bindings-x-lang/index>` provides a few
+pointers on how you may want to approach this.
+
+You can also use the :doc:`HTTP JSON API Service </json-api/index>` to tap into the Ledger API.
+
+At its core, it provides a simplified view of the active contract set and additional primities to query it and
+exposing it using a well-defined JSON-based encoding over a conventional HTTP connection.
+
+A subset of the services mentioned above is also available as part of it.
```

Comment:
As part of what?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/13271/commits/ac92165a493ddc851efe425f173e6c31c0d8ecf7